src/eko/runner/parts.py

```diff
@@ -88,6 +88,7 @@ def evolve_configs(eko: EKO) -> dict:
         n_integration_cores=ocard.configs.n_integration_cores,
         ModSV=ocard.configs.scvar_method,
         n3lo_ad_variation=tcard.n3lo_ad_variation,
+        matching_order=tcard.matching_order,
```

Comment:
```suggestion
        matching_order=tcard.matching_order if tcard.matching_order is not None else tcard.order,
```

I always wondered if there is an easier way for the pattern `x if x is not None else y`. But the only thing it comes to my mind is `x or y`, and it is bad because for `x = False` it would return `y`...

Reply:
Actually here I think we have introduced a bug.
`matching_order`  follows `PTO` (ie is 2 for NNLO) so here we should subtract one, once doing the default, no?

Reply:
just to have easier tracking: moved to #322